test/MusicStore.Spa.Test/ShoppingCartTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Http.Core.Collections;
+using Xunit;
+
+namespace MusicStore.Models
+{
+    public class ShoppingCartTest
```

Comment:
Adding @DamianEdwards since I'm adding unit tests for the spa version of MusicStore. 

There is no actions using Cookies in MusicStore and MusicStore.Spa sample. The only place is the ShoppingCart in MusicStore.Spa. (The ShoppingCart in MusicStore uses Session). Plus, even any of the actions/controllers in MusicStore.Spa doesn't use its ShoppingCart. So, I wrote a unit test directly for ShoppingCart for Cookies.